starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,14 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
 
-
+  articleController.init = function() {
+    module.Article.fetchAll(module.articleView.initIndexPage);
+    $('.tab-content').show();
```

Comment:
So you are showing both the articles section and the about section. You need to hide all the sections, then show just the articles section.